README.md

```diff
@@ -3,10 +3,10 @@
 *GeoTrellis* is a Scala library and framework for creating processes to work with raster data.
 
 #### IO:
-GeoTrellis reads, writes, and operates on raster data as fast as possible. It also has functionality to warp (change the resolution and bounding box of) rasters on loading and throughout the operation sequence.
+geoTrellis reads, writes, and operates on raster data as fast as possible. It also has functionality to warp (change the resolution and bounding box of) rasters on loading and throughout the operation sequence.
```

Comment:
Please use capital letters in the **IO** section.  The **Operations** section is fine :-)